trade-finance-manager-ui/server/controllers/case/amendments/validation/amendmentYearValidation.validate.test.js

```diff
@@ -0,0 +1,45 @@
+const amendmentmentYearValidation = require('./amendmentYearValidation.validate');
+
+describe('amendmentmentYearValidation()', () => {
+  it('should return true when year is 2 numbers long', () => {
+    const year = '22';
+    const result = amendmentmentYearValidation(year);
```

Comment:
Can we please add two more cases one with `1` number and another with `3` number please.
Otherwise good coverage here.

Reply:
done